department-of-veterans-affairs/va.gov-cms
# VACMS-4371 vc satellite facility

## Description
See #4371 

## Testing done
Visual, behat, phpunti

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/108751184-e876ef80-750f-11eb-90e2-dc9002e2dab4.png)
![image](https://user-images.githubusercontent.com/2404547/108751199-ec0a7680-750f-11eb-84db-1f19072948d7.png)

## QA steps
- As an administrator, go to `node/add/vet_center_satellite_facility` and visually verify form appears as in above form screenshot.
- Fill in the fields, save, and visually verify form and path saves correctly.

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
